mitodl/micromasters
# Return course certificate url only if course has signatories

#### What are the relevant tickets?
Related to this comment https://github.com/mitodl/micromasters/pull/3546#issuecomment-332222750

#### What's this PR do?
Changes `get_certificate_url` for course to return the certificate url only if the course has `CourseCertificateSignatories`.

#### How should this be manually tested?
When you have a certificate for a course and there are signatories for same course, you should be able to see the 'View Certificate' link in the dashboard.
